src/main/java/reactor/netty/resources/ConnectionProvider.java

```diff
@@ -240,13 +242,29 @@ default int maxConnections() {
 		return -1;
 	}
 
+	/**
+	 * Create a {@link ConnectionProvider}.
+	 * <ul>
+	 *     <li>
+	 *         Elastic {@link ConnectionProvider} will cache and grow on demand {@link Connection}.
+	 *         <p>It will never wait before opening a new connection.
+	 *         The reuse window is limited but it cannot starve an undetermined volume of clients using it.
+	 *     </li>
+	 *     <li>
+	 *         Fixed {@link ConnectionProvider} will cache and reuse a fixed maximum number of {@link Connection}.
+	 *         <p>It will open up to the given max number of processors observed by this jvm (minimum 4).
+	 *         Further connections will be pending acquisition depending on acquireTimeout.
+	 *     </li>
+	 * </ul>
+	 */
```

Comment:
This would become simpler if there was no fixed vs elastic.

Reply:
done

Reply:
done